x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/util/Check.java

```diff
@@ -36,4 +36,14 @@ public static void notNull(Object object, String message, Object... values) {
             throw new SqlIllegalArgumentException(message, values);
         }
     }
+    
+    public static void isNumberOutOfRange(Object object, String objectName, Long from, Long to) {
```

Comment:
The `Check` class' semantics seem to hint a naming in the lines of "check that <predicate>" (Check#isNull, Check#isTrue). Also, the function doesn't handle floating point `Number`s.
```suggestion
    public static void isFixedNumberAndInRange(Object object, String objectName, Long from, Long to) {
```

Reply:
Thanks for the advice.